cli/cmd/compliance_aws.go

```diff
@@ -84,6 +84,10 @@ Then navigate to Settings > Integrations > Cloud Accounts.
 		Use:     "get-report <account_id>",
 		Aliases: []string{"get"},
 		PreRunE: func(_ *cobra.Command, _ []string) error {
+			if compCmdState.Csv {
+				cli.NonInteractive()
+			}
```

Comment:
Does interactive affect the csv output?

Reply:
Yes, with the CSV output we do not want any of the spinners/upgrade-notes/etc to output, so this disables those.

Reply:
I have a feeling that you are trying to pipe the CSV output to a file `lacework comp ... > file.csv` which in that case, the user should use the global flag `--noninteractive`.

Are we trying to download the file like the PDF flag?